### ISSUE TITLE
White Balance presets DSC-RX100M7

### ISSUE THREAD

**Issue by itkoala56** [darktable-org/darktable#12923]
a little work for White Balance presets DSC-RX100M7
[DSC-RX100M7-darktable-whitebalance-20221116.tar.gz](https://github.com/darktable-org/darktable/files/10023438/DSC-RX100M7-darktable-whitebalance-20221116.tar.gz)

**Comment by TurboGit** [comment]
I'm not sure what should be done of all those .txt in this archives. The white-balance have been moved into JSON and must be added there with a proper PR. TIA.

**Comment by itkoala56** [comment]
I followed this guide
https://github.com/darktable-org/darktable/wiki/White-balance-presets
The script extract_wb_from_images.sh has created this file with the text files
and I quote from this guide
"A developer will take care of the data and create presets in darktable"

Manwhile  I built myself a development environment, and tried the way with
extract_wb.py
[add on for wb_presets.zip](https://github.com/darktable-org/darktable/files/10102229/add.on.for.wb_presets.zip)

The result was the content of the attached file
When running the script I received 29 warnings.
WARNING: DSC04600.ARW has finetuning over GM axis! Data is skewed!
ans so on
The results from the photos with warnings are not included in this file.
